package.json

```diff
@@ -12,18 +12,22 @@
   "devDependencies": {
     "@babel/core": "^7.1.0",
     "@babel/plugin-proposal-class-properties": "^7.1.0",
+    "@babel/plugin-transform-runtime": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
-    "babel-loader": "^8.0.2",
-    "body-parser": "^1.18.3",
-    "classnames": "^2.2.6",
+    "babel-eslint": "^10.0.1",
+    "babel-loader": "^8.0.4",
     "css-loader": "^1.0.0",
-    "express": "^4.16.3",
+    "eslint": "^5.9.0",
+    "eslint-config-airbnb": "^17.1.0",
+    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-jest": "^22.1.0",
+    "eslint-plugin-jsx-a11y": "^6.1.2",
+    "eslint-plugin-react": "^7.11.1",
+    "event-stream": "3.3.4",
```

Comment:
What is that event-stream for?

Reply:
It's older version of this package due to security vulnerabilities. It's here temporary I think.

Reply:
You should try to find the package that has this vulnerability and fix it. There is no point in adding dependencies that are not directly needed by you

Reply:
But how to fix it if the latest version of this package which has a dangerous code installs automatically through "npm install"?

Reply:
Which package is that?

Reply:
Ok I read that version is lock to 3.3.4 so I deleted it from dev dep.